### PR TITLE
[BB-446] fix: Make form field labels consistent in Logos page

### PR DESCRIPTION
**Pull request by kewne**
This adds pruned down versions of `ImageUploadField` and `TextInputField` that are easier to style.
It also makes some style tweaks to make things more consistent.

Related tickets:
* [BB-4446 (OpenCraft Internal)](https://tasks.opencraft.com/browse/BB-4446)

# Testing

1. Go to "Logo" section in console;
2. Verify that all input heading are consistent.

# Screenshot

image

### CODE REVIEW

File: frontend/.eslintrc.json [hunk 1]
```diff
@@ -44,6 +44,7 @@
       "import/no-extraneous-dependencies": ["error", {"devDependencies": true}],
       "max-len": ["error", {"comments": 100, "code": 130}],
       "import/no-unresolved": ["error",{"ignore": [".scss"]}],
-      "react/jsx-props-no-spreading": ["off"]
+      "react/jsx-props-no-spreading": ["off"],
+      "react/jsx-wrap-multilines": ["error", {"prop": "ignore"}]
```
Comment (kewne) [inline frontend/.eslintrc.json:48]: I added this so things like the below are acceptable:
```jsx
<Component
  label={
    <div>
      <span>Ho ho ho</span>
    </div>
  }
/>
```

Without this, the linter complains about about missing parens but, when you add them, a different rule complains that they are unnecessary.

File: frontend/src/ui/components/ImageUploadField/ImageUploadField.tsx [hunk 2]
```diff
@@ -5,44 +5,46 @@ import upArrowIcon from 'assets/uparrow.png';
...
-  children?: React.ReactNode;
 }
 
 interface Image {
   name?: string;
 }
 
-export const ImageUploadField: React.FC<ImageUploadFieldProps> = (
-  props: ImageUploadFieldProps
+export const ModalImageInput: React.FC<ModalImageInputProps> = (
```
Comment (kewne) [inline frontend/src/ui/components/ImageUploadField/ImageUploadField.tsx:29]: This component is an extraction of just the actual input parts of `ImageUploadField`.

File: frontend/src/ui/components/ImageUploadField/ImageUploadField.tsx [hunk 3]
```diff
@@ -201,6 +169,68 @@ export const ImageUploadField: React.FC<ImageUploadFieldProps> = (
...
   );
 };
+
+interface OverlayTooltipProps {
+  id: string;
+  tooltipImage?: string;
+  innerPreview?: string;
+  children: React.ReactNode;
+}
+export const OverlayTooltip: React.FC<OverlayTooltipProps> = props => {
```
Comment (kewne) [inline frontend/src/ui/components/ImageUploadField/ImageUploadField.tsx:218]: This is an extraction of the Tooltip logic from `ImageUploadField`.

File: frontend/src/ui/components/TextInputField/TextInputField.tsx [hunk 4]
```diff
@@ -21,15 +21,46 @@ interface InputFieldProps {
...
+  label: React.ReactNode;
+  loading?: boolean;
+  onBlur?: React.FocusEventHandler<HTMLInputElement>;
+  onChange?: React.ChangeEventHandler<HTMLInputElement>;
+  reset?: Function;
+  type?: string;
+  value?: string;
+}
+
+export const TextInputField2: React.FC<TextInputField2Props> = (
```
Comment (kewne) [inline frontend/src/ui/components/TextInputField/TextInputField.tsx:57]: Again, an extraction of `TextInputField` that allows custom content in the label.

File: frontend/src/styles/_global.scss [hunk 5]
```diff
@@ -27,3 +27,7 @@ body {
   border: 1px solid $secondary-1;
   background-color: $secondary-1;
 }
+
+.text-bold {
+  font-weight: bold;
+}
```
Comment (0x29a) [inline frontend/src/styles/_global.scss:33]: Nit: it's not necessary to introduce a new class. [Boostrap built-in classes](https://getbootstrap.com/docs/4.0/utilities/text/#font-weight-and-italics) can be used.

**Review by 0x29a — APPROVED** [review]
:+1:

- [x] I tested that logos page now has consistent style.
- [x] I read through the code.

File: frontend/src/styles/_global.scss (hunk 5 above)
Comment (kewne) [reply frontend/src/styles/_global.scss]: Done!

**Pull request merged**